cpp/example_code/cross-service/serverless-aurora/serverless_aurora.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+/**
+ *  serverless_aurora.cpp
+ *
+ *  The code in this file contains a main routine for running the example as an application.
+ *
+ * To run the example, refer to the instructions in the README.
+ *
+ */
+
+#include "ItemTrackerHTTPHandler.h"
+#include "RDSDataHandler.h"
+#include "SES3EmailHandler.h"
+#include "PocoHTTPServer.h"
+
+static const Aws::String TABLE_NAME("items");
+
+//! Routine which runs Amazon Aurora Serverless as an HTTP server using the Poco library.
```

Comment:
-> ...runs the Amazon Aurora Serverless example as...
(otherwise it sounds like you're running Aurora itself)